public/app/features/dashboard/components/TransformationsEditor/TransformationPicker.tsx

```diff
@@ -0,0 +1,122 @@
+import { css } from '@emotion/css';
+import React, { FormEventHandler, KeyboardEventHandler, ReactNode } from 'react';
+
+import { DocsId, GrafanaTheme2, TransformerRegistryItem } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
+import { Card, Container, VerticalGroup, Alert, Input, useStyles2 } from '@grafana/ui';
+import { LocalStorageValueProvider } from 'app/core/components/LocalStorageValueProvider';
+import { getDocsLink } from 'app/core/utils/docsLinks';
+import { PluginStateInfo } from 'app/features/plugins/components/PluginStateInfo';
+
+const LOCAL_STORAGE_KEY = 'dashboard.components.TransformationEditor.featureInfoBox.isDismissed';
+
+interface TransformationPickerProps {
+  noTransforms: boolean;
+  search: string;
+  onSearchChange: FormEventHandler<HTMLInputElement>;
+  onSearchKeyDown: KeyboardEventHandler<HTMLInputElement>;
+  onTransformationAdd: Function;
+  suffix: ReactNode;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Any way around this `any`?

Reply:
Been thinking about this one. The reason that it's _any_ is that transformations can have, well, _any_ structure for their options 😂 . This type had been like this before, and we can look at typing it out but it would probably require a fair amount of in-depth effort.